thumbor/Dockerfile

```diff
@@ -19,20 +19,20 @@ ENV SHELL bash
 ENV WORKON_HOME /usr/src/app
 WORKDIR /usr/src/app
 
-ARG DOCKERHOST=172.17.0.1
-ENV DOCKERHOST ${DOCKERHOST}
 COPY requirements.txt /usr/src/app/requirements.txt
-RUN pip install --trusted-host None --no-cache-dir --use-wheel \
-   --extra-index-url http://${DOCKERHOST}:9009/simple/ \
-   --trusted-host ${DOCKERHOST} \
+RUN pip install --trusted-host None --no-cache-dir \
    -r /usr/src/app/requirements.txt
 
 COPY conf/thumbor.conf.tpl /usr/src/app/thumbor.conf.tpl
 
-RUN \ 
+RUN \
     ln /usr/lib/python2.7/dist-packages/cv2.x86_64-linux-gnu.so /usr/local/lib/python2.7/cv2.so && \
     ln /usr/lib/python2.7/dist-packages/cv.py /usr/local/lib/python2.7/cv.py
 
+ARG SIMD_LEVEL
+# pinning to 4.3.0 see https://github.com/uploadcare/pillow-simd/issues/17
+RUN if [ -n "$SIMD_LEVEL" ]; then pip uninstall -y pillow; CC="cc -m$SIMD_LEVEL" pip install --no-cache-dir -U --force-reinstall Pillow-SIMD==4.3.0.post0; fi
```

Comment:
I think there is a way to use same version of pillow-simd as pillow. Something like this:
```bash
if [ -n "$SIMD_LEVEL" ]; then \
    PILLOW_VERSION=$(python -c 'import PIL; print(PIL.__version__)') && \
    pip uninstall -y pillow || true && \
    CC="cc -m$SIMD_LEVEL" pip install --no-cache-dir -U --force-reinstall "pillow-simd<=$PILLOW_VERSION" \
    fi
```

Reply:
Thanks @kkopachev this sounds like a great idea. 

I still wonder though, do the versions match 100%? because I can see that Pillow would likely use [`4.3.0`](https://pypi.org/project/Pillow/#history) but Pillow-SIMD would be using [`4.3.0.post0`](https://pypi.org/project/Pillow-SIMD/#history), so versions might still mismatch slightly?